builtin/logical/ssh/path_config_ca_test.go

```diff
@@ -167,4 +168,132 @@ func TestSSH_ConfigCAUpdateDelete(t *testing.T) {
 	if err != nil || (resp != nil && resp.IsError()) {
 		t.Fatalf("bad: err: %v, resp:%v", err, resp)
 	}
+
+	// Delete the configured keys
+	caReq.Operation = logical.DeleteOperation
+	resp, err = b.HandleRequest(context.Background(), caReq)
+	if err != nil || (resp != nil && resp.IsError()) {
+		t.Fatalf("bad: err: %v, resp:%v", err, resp)
+	}
+
+	// Successfully create a new RSA/2048 one
```

Comment:
Could you convert this to a table driven test to cut down on the overall code and make it clearer in the future if a test fails which algo is causing us issues?